src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/UserDefinedImplicitConversions.cs

```diff
@@ -634,6 +635,7 @@ private static bool IsEncompassingImplicitConversionKind(ConversionKind kind)
                 case ConversionKind.Boxing:
                 case ConversionKind.ImplicitConstant:
                 case ConversionKind.ImplicitPointerToVoid:
+                case ConversionKind.ImplicitPointer:
```

Comment:
>case ConversionKind.ImplicitPointer: [](http://example.com/codeflow?start=16&length=36)

Does this fall into the C# 4 category as the comment above suggests?

Reply:
No, I'll move it. I should also update the spec to include that this is in the set of standard implicit conversions, as it's technically not there at the moment (but was implied as all type-to-type conversions that aren't about expressions or dynamic are).